examples/profile.php

```diff
@@ -17,7 +17,14 @@
     $base64Selfie = ActivityDetailsHelper::getBase64Selfie($profile);
     // This key uses the  format: age_[over|under]:[1-999] and is dynamically
     // generated based on the dashboard attribute Age / Verify Condition
-    $ageVerified = $profile->getProfileAttribute('age_over:18');
+    $ageVerified = $profile->isAgeVerified();
+    if(is_bool($ageVerified)) {
```

Comment:
This conditional doesn't look alright to me. Let's look into it a bit more.

Reply:
This function will return two types a boolean `TRUE or FALSE` if the attribute is set or a string `NULL` if the attribute is not set.